src/components/PrizePage/Prizes.js

```diff
@@ -81,35 +121,66 @@ export default function Prizes() {
 	const hideImg = useMediaQuery(theme.breakpoints.down('xs'));
 
 	return (
-		<Container maxWidth='md'>
-			<TableContainer>
-				<Table aria-label="prize table">
-					<TableHead>
-						<TableRow>
-							{ hideImg ? null : <TableCell></TableCell> }
-							<TableCell align='center' className={classes.header}>Track</TableCell>
-							<TableCell align='center' className={classes.header}>Prize</TableCell>
-						</TableRow>
-					</TableHead>
-					<TableBody>
-						{allPrizes.map(row =>
-							<TableRow key={row.track}>
-								{ hideImg ?
-									null :
-									<TableCell align='center' width='20%'>
-										<img src={row.img} height='60px'/>
+		<>
+			<Container maxWidth='md'>
+				<Grid maxWidth='md' container spacing={3} className={classes.prizeGrid}>
+					{allPrizes.map(prize =>
+						<Grid item xs={12} sm={4} key={prize.track}>
+							<Box className={classes.prizeBox}>
+								<Box className={classes.imageBox}>
+									<img src={prize.img} height='60px'/>
+								</Box>
+								<Box className={classes.prizeCard}>
+									<Typography variant='h6' component='h3'>
+										{prize.track}
+									</Typography>
+									<Box paddingLeft='20%' paddingRight='20%'>
```

Comment:
nit:
```suggestion
									<Box paddingX='20%'>
```

check https://material-ui.com/system/spacing/